openage/convert/processor/export/media_exporter.py

```diff
@@ -122,6 +123,12 @@ def _export_blend(export_request, sourcedir, exportdir, blend_mode_count=None):
                 exportdir[export_request.targetdir],
                 f"{export_request.target_filename}{idx}.png"
             )
+        source_format = export_request[export_request.source_filename.rindex("."):]
```

Comment:
Export request has no operation for `[]`, so `export_request[export_request.source_filename.rindex("."):]` doesn't work. I guess you are trying to get the extension of the filename. There is a quicker way to access it is using the `suffix` property of the source file path stored in `source_file`, see

https://github.com/SFTtech/openage/blob/master/openage/util/fslike/path.py#L265-L271

The same mechanism exists in Python's `pathlib` (https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.suffix)